src/styles/partials/components/_explore.scss

```diff
@@ -0,0 +1,56 @@
+.dg_explore-link {
+  background: $gray-lightest;
+  display: block;
+  font-family: $default-heading-font;
+  font-weight: 600;
+  font-size: $smallest-heading-font-size;
+  min-height: 130px;
+  margin-bottom: $default-margin;
+  padding: $padding-larger;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  &.focused,
+  &:focus {
+    color: $blue;
+  }
+
+  &:hover,
+  &.hovered {
+    background: $vibrant-blue;
+    color: $white;
+  }
+
+  &:active,
+  &.active {
+    background: $pale-blue;
+    color: $white;
+  }
+}
+
+.dg_explore {
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+
+  .dg_explore-link {
+    margin-right: $default-margin;
+    width: 30%;
+  }
+}
+
+@media screen and (max-width: $medium-breakpoint) {
```

Comment:
Between 769px and 897px, the items appear to stack two across. May consider adding a larger breakpoint.

Reply:
will take a look.

Reply:
@theilman7 per our dicussion, this is looking ok i think, please just double check on yoru machine.

Reply:
Closing issue. Will log a bug if the issue shows up when we view on template. @martypowell